resources/views/teacher/dashboard.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="block-header">
+    <div class="row clearfix mb-2">
+        <div class="col-md-6 col-sm-12">
+            <h1>Dashboard</h1>
+            <nav aria-label="breadcrumb">
+                <ol class="breadcrumb">
+                <li class="breadcrumb-item"><a href="{{ route('dashboard') }}">Learnify.id</a></li>
+                <li class="breadcrumb-item active" aria-current="page">Dashboard</li>
+                </ol>
+            </nav>
+        </div>
+    </div>
+    <div class="row-clearfix mt-5">
+        <h1 class="color-blue-2 font-weight-bold text-uppercase" style="font-size: 1.8rem;">matematika</h1>
+        <div class="d-flex justify-content-between align-items-top mt-3 font-weight-bold">
+            <p>Terdapat <span class="color-blue-2">100</span> Materi!</p>
+            <a href="{{ route('subject') }}" class="color-blue-2">Lihat Semua</a>
+        </div>
+        <div class="mt-3">
```

Comment:
```suggestion
    <div class="row-clearfix mt-3">
        <h5 class="color-blue-2 font-weight-bold text-uppercase">matematika | xii</h5>
        <div class="d-flex justify-content-between mt-3 align-items-end font-weight-bold">
            <a class="color-black">Terdapat <span class="color-blue-2">100</span> Materi!</a>
            <a href="{{ route('subject') }}" class="color-blue-2">Lihat Semua</a>
        </div>
        <div class="mt-1">
```
sama disamping button lihat semua tambahin icon chevron right dari fontawesome